charts/spire/charts/spire-agent/values.yaml

```
@@ -170,6 +170,17 @@ workloadAttestors:
     ## @param workloadAttestors.k8s.disableContainerSelectors Set to true if using holdApplicationUntilProxyStarts in Istio
     disableContainerSelectors: false
 
+# Optional SDS configuration section.
+sds:
```

```suggestion
sds:
  ## @param sds.enabled Enables Envoy SDS configuration
  enabled: false
```

Don't forget to regenerate the docs.